lib/phobos.rb

```diff
@@ -77,8 +77,8 @@ def create_exponential_backoff(backoff_config = nil)
       ExponentialBackoff.new(min, max).tap { |backoff| backoff.randomize_factor = rand }
     end
 
-    def deprecate(message)
-      warn "DEPRECATION WARNING: #{message} #{Kernel.caller.first}"
+    def deprecate(message, location)
+      warn "DEPRECATION WARNING: #{message}: #{location}"
```

Comment:
If you're trying to avoid printing out Phobos methods, this is probably a better generalized solution (note you can omit the "Kernel" part of the call):
`caller.find { |s| s !~ /\/phobos\// }`
See if that works, in which case you don't need to change all the other calling code.

Reply:
Yep, this does work 👍 Updated the code accordingly!